src/fontAwesomeIcon.js

```diff
@@ -0,0 +1,66 @@
+let redact;
+
+const FONT_AWESOME_ICON_RE = /^<i class="fa fa-([a-z\-]+)" aria-hidden="true"><\/i>/;
+const FONT_AWESOME_ICON = 'fontAwesomeIcon';
+
+/**
+ * A very basic plugin whose only purpose is to redact (and restore) Font
+ * Awesome icons.
+ */
+module.exports = function fontAwesomeIcon() {
+  if (this.Parser) {
```

Comment:
where does `this.Parser` get set?

Reply:
in Remark; it's part of the whole structure of plugins. See https://github.com/unifiedjs/unified#processorparser for more details